src/main/resources/assets/sass/main.scss

```diff
@@ -315,3 +315,13 @@ dl {
 pre {
     margin: 0;
 }
+
+.open-government-licence {
+	.logo {
```

Comment:
Indentation looks odd here. Should be 2 spaces

Reply:
Addressed.